src/euphorie/deployment/locales/it/LC_MESSAGES/euphorie.po

```diff
@@ -6411,29 +6800,24 @@ msgstr "Descrizione del rischio"
 msgid "report_heading_situation"
 msgstr "Situazione"
 
-#. Default: "This document was based on the OiRA Tool '${title}' of revision date ${date}."
+#. Default: "This document was based on the OiRA Tool '${title}' of revision
+#. date ${date}."
 #: euphorie/client/docx/compiler.py:991
 msgid "report_identification_revision"
 msgstr ""
 "Questo documento è stato basato sulla data di revisione ${date} del tool "
 "OiRA '${title}'."
 
-#. Default: "Now that you have finished identifying all the risks in your organisation and created an action plan you can create a report which lists all your results. You can, if desired, add some extra comments that should be included in this report in the field below."
+#. Default: "Now that you have finished identifying all the risks in your
+#. organisation and created an action plan you can create a report which lists
+#. all your results. You can, if desired, add some extra comments that should
+#. be included in this report in the field below."
 #: euphorie/client/browser/templates/report.pt:34
 msgid "report_intro"
 msgstr ""
-"<p>A seguito della identificazione dei pericoli, della valutazione dei "
-"rischi e, dopo aver adottato le misure obbligatorie e programmato le misure "
-"di miglioramento, si genera progressivamente il documento di valutazione dei "
-"rischi.</p>\n"
-"<p>ATTENZIONE: PRIMA DI STAMPARE LA VERSIONE FINALE DEL REPORT, CONTROLLARE "
-"DI AVER IDENTIFICATO E VALUTATO TUTTI I RISCHI PRESENTI; E’ NECESSARIO "
-"ALLEGARE IL DOCUMENTO CON LE FIGURE DELLA PREVENZIONE E TUTTI GLI ALTRI "
-"DOCUMENTI CITATI NEL TESTO, APPOSITAMENTI COMPILATI, LADDOVE NECESSARI.</p>\n"
-"<p>Il documento deve essere munito di “data certa” o attestata dalla "
-"sottoscrizione dello stesso, da parte del datore di lavoro nonché ai soli "
-"fini della prova della data, da parte di RSPP, RLS o RLST e del Medico "
-"Competente, ove nominato.</p>"
+"<p>A seguito della identificazione dei pericoli, della valutazione dei rischi e, dopo aver adottato le misure obbligatorie e programmato le misure di miglioramento, si genera progressivamente il documento di valutazione dei rischi.</p>\n"
```

Comment:
Is this `<p>` right?

Reply:
I assume the italian partner asked to make the ATTENZIONE part stand out. If there are no p's, the uppercase text gets swallowed by the rest.

![Screenshot 2025-04-02 at 08 54 31](https://github.com/user-attachments/assets/36857756-546d-49be-9eaf-0140eed0c2e6)

versus

![Screenshot 2025-04-02 at 08 56 28](https://github.com/user-attachments/assets/eaf1b90f-6dee-4239-90e9-d4dc66e4120c)